src/components/RentalFees/RentalFeeTs.ts

```diff
@@ -0,0 +1,35 @@
+import { Component, Vue, Prop, Watch } from 'vue-property-decorator'
+//@ts-ignore
+import MosaicAmountDisplay from '@/components/MosaicAmountDisplay/MosaicAmountDisplay.vue'
+//@ts-ignore
+import FormRow from '@/components/FormRow/FormRow.vue'
+
+export type RentalFeesType = 'mosaic' | 'root-namespace' | 'child-namespace'
+@Component({
+  components: {
+    MosaicAmountDisplay,
+    FormRow,
+  },
+})
+export class RentalFeeTs extends Vue {
```

Comment:
recommendation to use store getters instead of statically passed in `rentalFees` component-level property